### PR TITLE
Style check "How to Shield Your Linux Resources"

### PR DESCRIPTION
**Pull request by tomschr**
@JeffreyCheung: I've run our style checker and it found several issues. I mostly focused on these things:

* Try to simplify overly long sentences
* Reduce/remove wordy phrases

Let me know if this is still ok.

### CODE REVIEW

**Comment by JeffreyCheung** [comment]
I am ok of your work and thanks

File: xml/slert_shielding_model.xml [hunk 1]
```diff
@@ -392,7 +412,7 @@ alext    14241 14212 Roth python ./cset shield -s -v</screen>
      Use of the appropriate PIDSPEC can thus be handy to move tasks and
      groups of tasks into and out of the shield. Additionally, there is one
      more option that can help with multi-threaded processes, and that is
-     the <option>--threads</option> flag. If this flag is present in a
+     the <option>--threads</option> flag. If this flag is be in a
```
Comment (lpechacek) [inline xml/slert_shielding_model.xml]: As above:  "If this flag is be in ..."

File: xml/slert_usingshortcuts.xml [hunk 2]
```diff
@@ -178,9 +178,9 @@
 
   <para>
    The <command>set</command> subcommand has a limited number of shortcuts.
-   Basically, the <option>-s</option>/<option>--set</option> option is optional in most cases
-   and the <option>-l/</option>/<option>--list</option> option is also optional if you want to
-   list sets. For example, these commands are equivalent.
+   The options <option>-s</option>/<option>--set</option> are optional usually
```
Comment (lpechacek) [inline xml/slert_usingshortcuts.xml]: I would suggest singular for "option" again.

File: xml/slert_cpuset_manipulation.xml [hunk 3]
```diff
@@ -483,7 +482,7 @@ cset: "two" cpuset of CPUSPEC(2) with 0 tasks running</screen>
     Use of the appropriate PIDSPEC can thus be handy to move tasks and
     groups of tasks. Additionally, there is one more option that can help
     with multi-threaded processes, and that is the
-    <option>--threads</option> flag. If this flag is present in a
+    <option>--threads</option> flag. If this flag is be in a
```
Comment (tomschr) [inline xml/slert_cpuset_manipulation.xml]: I like your suggestion, implemented.

File: xml/slert_cpuset_manipulation.xml [hunk 4]
```diff
@@ -856,13 +855,13 @@ system       0 n          0 n       257   0    /system</screen>
   <para>
    There is a planned addition to the <command>proc</command> subcommand
    that will allow moving a specified PIDSPEC of tasks running in a
-   specified cpuset to its parent. This addition will ease the automation
+   specified cpuset to its parent. This addition will simplify the automation
```
Comment (tomschr) [inline xml/slert_cpuset_manipulation.xml]: Ok, I'll disable them for the time being.

File: xml/slert_shielding_model.xml [hunk 5]
```diff
@@ -4,83 +4,103 @@
...
  <sect1 xml:id="sec.shielding.model.ex">
   <title>A Simple Shielding Example</title>
-
-  <para>
-   Assume that we have a 4-way machine that is not NUMA. This means there
-   are 4 CPUs at our disposal and there is only one Memory Node available.
-   On such machines, we do not need to specify any memory node parameters to
-   cset, it sets up the only available memory node by default.
-  </para>
+  <para> Assume that we have a 4-core machine that is not NUMA. This means
```
Comment (tomschr) [inline xml/slert_shielding_model.xml]: Ok, done.

However, NUMA appears in the files `art_slert_quickstart.xml`, `art_slert_virtguide.xml`, `slert_cpuset_manipulation.xml`, and `slert_intro.xml`. Should they replaced too?

File: xml/slert_usingshortcuts.xml (hunk 2 above)
Comment (tomschr) [inline xml/slert_usingshortcuts.xml]: I used only `--set` and `--list` as in my previous example to avoid the singular/plural mess.

**Comment by tomschr** [comment]
Thanks for the review, I've added all your changes and merged it into develop branch.

File: xml/slert_shielding_model.xml (hunk 5 above)
Comment (lpechacek) [inline xml/slert_shielding_model.xml]: Not completely sure about the question.  Looking at SLERT documentation I can see proper use of the term "NUMA".  It is only "non-NUMA" which is evil as it expands into "non-Non Uniform Memory Access" which is a double negation. So no more changes needed here from my POV.

**Pull request closed without merging**